Kernel/Arch/aarch64/init.cpp

```diff
@@ -102,7 +102,7 @@ extern "C" [[noreturn]] void __stack_chk_fail();
 
 void __stack_chk_fail()
 {
-    Prekernel::halt();
+    Kernel::Processor::halt();
```

Comment:
Do we need `Kernel::` here given the `using namespace`?

Reply:
For whatever reason the `using namespace` is below this function, and it only affects code after the `using namespace` decl. The `init` function and code in this file should really be living in the `Kernel` namespace anyway, so I added an commit that does that!